Add Capital Whiterun Expansion

robertgk2017 opened the pull request:
Add Capital Whiterun Expansion

and

Add Rob's Bug Fixes - Capital Whiterun Expansion update message

robertgk2017 committed:
Add Capital Whiterun Expansion

Also add my Rob's Bug Fixes replacer .esp as an update file message

robertgk2017 committed:
Update masterlist.yaml

sibir-ine commented on masterlist.yaml:
```diff
@@ -25514,3 +25514,19 @@ plugins:
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/37982/'
+        name: 'Capital Whiterun Expansion'
+        link: 'https://www.nexusmods.com/skyrimspecialedition/mods/63355/'
```
Additional links need to be their own elements:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/63355/'
```

robertgk2017 committed:
Removed redundant link

sibir-ine commented on masterlist.yaml:
```diff
@@ -25514,3 +25514,17 @@ plugins:
+    clean:
+      - crc: 0xD96CF5BF
+        util: 'SSEEdit v4.0.4b'
+        crc: 0x538498A4
```
```suggestion
      - crc: 0x538498A4
```

robertgk2017 committed:
Update masterlist.yaml

robertgk2017 committed:
Update masterlist.yaml

Co-authored-by: sibir <ine.sibir@gmail.com>

robertgk2017 committed:
Update masterlist.yaml

robertgk2017 committed:
Added data for my included fishing patch.

sibir-ine commented on masterlist.yaml:
```diff
@@ -25514,3 +25514,32 @@ plugins:
+        util: 'SSEEdit v4.0.4b'
+      - crc: 0x538498A4
+        util: 'SSEEdit v4.0.4b'
+
```
A separating newline isn't necessary when they're from the same page:

```suggestion
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -25514,3 +25514,32 @@ plugins:
+
+  - name: 'SurWR - Fishing.esp'
+    url:
+      - link: https://www.nexusmods.com/skyrimspecialedition/mods/63355/
```
The link needs to be single-quoted:

```suggestion
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/63355/'
```

sibir-ine commented on masterlist.yaml:
```diff
@@ -25514,3 +25514,32 @@ plugins:
+    clean:
+      - crc: 0x28A22265
+        util: 'SSEEdit v4.0.4b'
+    after:
+      - 'SurWR.esp'
```
The patch has it as a master, so LOOT will automatically load them correctly.

```suggestion
```

robertgk2017 committed:
Update masterlist.yaml

removed the after key on the patch and conditioned the patchIncluded key for hte fshing patch

robertgk2017 committed:
Update masterlist.yaml

Co-authored-by: sibir <ine.sibir@gmail.com>

robertgk2017 committed:
Update masterlist.yaml

Co-authored-by: sibir <ine.sibir@gmail.com>

robertgk2017 committed:
Re arranged msg keys

robertgk2017 committed:
line spacing

robertgk2017 committed:
i did a thing

sibir-ine commented on masterlist.yaml:
```diff
@@ -25514,3 +25514,29 @@ plugins:
         util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 3
         udr: 3
+        
```
There's some excess whitespace here:

```suggestion

```

robertgk2017 committed:
Update masterlist.yaml

Co-authored-by: sibir <ine.sibir@gmail.com>

sibir-ine commented on masterlist.yaml:
```diff
@@ -25514,3 +25514,28 @@ plugins:
         util: '[SSEEdit v4.0.4b](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 3
         udr: 3
+  - name: 'SurWR.esp'
```
Sorry about that, the suggestion wasn't supposed to remove the newline as well. I forgot that GH's suggestions can be a bit finicky at times:

```suggestion

  - name: 'SurWR.esp'
```

robertgk2017 committed:
Update masterlist.yaml

Co-authored-by: sibir <ine.sibir@gmail.com>

sibir-ine reviewed (approved):
Thanks!
